packages/botonic-react/tests/components/multichannel/multichannel-facebook.test.jsx

```diff
@@ -0,0 +1,80 @@
+import { MultichannelFacebook } from '../../../src/components/multichannel/multichannel-facebook'
+
+describe('Multichannel Facebook text converter', () => {
+  const multichannelFacebook = new MultichannelFacebook()
+  const button = {
+    key: '1',
+    props: {
+      children: 'Children',
+      url: 'https://botonic.io/',
+    },
+    type: 'Button',
+  }
+  const shortText = [
+    'Continetur devenietur id ex denegassem ponderibus imaginatio patrocinio.',
+  ]
+  const longText = [
```

Comment:
what if the text is three times the max length? have you covered it?

Reply:
Yes it's covered. `longText` has almost 1400 characters, so in this case the text is divided in three messages